integrations/pytorch/recipes/sparse-transfer-learn.md

```diff
@@ -0,0 +1,98 @@
+<!--
```

Comment:
can we rename this to follow standards newly setup in the yolov5 integration? Specifically [model].transfer_learn_pruned.md and for model let's mark it as generic for classification by putting in 'classification' in place

Reply:
Resolved!